### ISSUE TITLE
NEED UPDATE

### ISSUE THREAD

**Issue by htmlgod** [madbyte/d2ric#4]
Plz add new items from 6.86 and 6.87! i can do it myself but i have a little time for this.

**Comment by madbyte** [comment]
hi,
i will take a look at it

**Comment by htmlgod** [comment]
thank you!

**Comment by Trini8ed** [comment]
I agree, honestly the only thing you could do is grab all the images off valves web servers and just have your program automatically pull all the data of the items from that and not have a need to update it. I would use the default one valve added in but you cant add recipes to the items...

**Comment by madbyte** [comment]
Hi,

I haven't had much time in the last weeks, but now I fixed some problems.
- All Heros should work now
- The Tool works with the reborn update
- All new items and recipes should work now

I need to add and change some item prices and tooltips

You can create the tool by yourself or wait until i done everything and upload a new version

**Comment by htmlgod** [comment]
THX U VERY MUCH!!

**Comment by madbyte** [comment]
You can find the update here: https://github.com/madbyte/d2ric/releases/latest

**Comment by Aq32** [comment]
Just a heads up that there will be a lot to update for: http://www.dota2.com/700

**Comment by madbyte** [comment]
I hope I have time next weekend to update everything. :)

**Comment by madbyte** [comment]
Items are updated.
Monkey King is still missing.
